app/build.gradle

```diff
@@ -72,7 +72,6 @@ android {
      * Update (both Gradle and local SDK) whenever possible
      */
     compileSdkVersion 27
-    buildToolsVersion '27.0.3'
```

Comment:
This shouldn't be set to `buildToolsVersion "28.0.3"`?

Reply:
See comment below 👍